weave/ops_arrow/arrow.py

```diff
@@ -182,31 +183,78 @@ def save_instance(self, obj, artifact, name):
                 obj._arrow_data, obj.object_type, obj._artifact, artifact
             )
 
-        from . import convert
+        # v1 AWL format
+        # from . import convert
+
+        # parquet_friendly = convert.to_parquet_friendly(obj)
+        # table = pa.table({"arr": parquet_friendly._arrow_data})
+        # with artifact.new_file(f"{name}.ArrowWeaveList.parquet", binary=True) as f:
+        #     pq.write_table(table, f)
```

Comment:
do we still support reading existing parquet files after this? or will we have to wipe the cache and transition over fully to feather?

Reply:
The read side implements both versions. So we don't need to wipe the cache.